ms2/src/org/labkey/ms2/showPeptide.jsp

```diff
@@ -66,7 +65,9 @@
 <%
         if (fraction.wasloadedFromGzFile())
         {
-            out.println(" " + MS2GZFileRenderer.getFileNameInGZFile(fraction.getFileName(), p.getScan(), p.getCharge(), "dta") + "<br>");
+            out.print(unsafe(" "));

```

Comment:
Would it make sense to introduce constants in HtmlString for normal spaces and <br> tags, like we have for non-breaking spaces? Seems like there are few places that are now doing unsafe() on them

Reply:
Yep, good idea. I don't want to overlap too much with DOM, but space and the most common tags would make sense.